playbooks/adhoc/uninstall.yml

```diff
@@ -55,6 +55,10 @@
     - pcsd
     failed_when: false
 
+  - shell: "atomic uninstall atomic-openshift-master"
+    changed_when: False
```

Comment:
@abutcher it looks like you were the fella to add this in ``playbooks/adhoc/uninstall.yml``. Do you recommend reusing ``changed_when: False`` in this kind of case?